### DISCUSSION TITLE
欲要求得身体的自由, 必先求得思想的自由

### DISCUSSION THREAD

**Discussion by Salted-Fish-Swimming** [The-Run-Philosophy-Organization/run#149] [General]
欲要求得身体的自由, 必先求得思想的自由. 中国的网络墙阻挡了求知的自由, 建议加一份有关翻墙的资料或者链接.

**Comment by EnderTheCoder** [comment]
现在github这网络环境。。。能上github基本都能翻墙

**Comment by the-run-philosophy** [comment]
有推荐的链接吗，可以引用一份进来

**Comment by Fodesu** [comment]
> 有推荐的链接吗，可以引用一份进来

https://github.com/bannedbook/fanqiang 这个怎么样，虽然要翻墙才能看到😅. 最近github裸连不了

**Comment by Theodoree** [comment]
我就挺好奇的,不翻墙怎么上的gayhub.

**Comment by ghost** [comment]
> > 有推荐的链接吗，可以引用一份进来
> 
> https://github.com/bannedbook/fanqiang 这个怎么样，虽然要翻墙才能看到😅. 最近github裸连不了

我个人感觉不太好，私货比较多（

**Comment by DJReact** [comment]
中国的网络墙阻挡了求知的自由
不敢苟同啊、墙外也就那样。半斤八两、
除了看p站方便

**Reply by getColder** [reply]
是啊，在哪都逃不过政治正确

**Reply by ghost** [reply]
> 中国的网络墙阻挡了求知的自由 不敢苟同啊、墙外也就那样。半斤八两、 除了看p站方便

https://m.cmx.im/@chenzhong/108214115356225623

**Reply by ghost** [reply]
> 是啊，在哪都逃不过政治正确

https://m.cmx.im/@taotao/108209596459197153
